bindings/azure/storagequeues/storagequeues.go

```diff
@@ -128,7 +146,12 @@ func (a *AzureStorageQueues) Init(metadata bindings.Metadata) error {
 	}
 	a.metadata = meta
 
-	err = a.helper.Init(a.metadata.AccountName, a.metadata.AccountKey, a.metadata.QueueName)
+	decodeBase64 := false
+	if a.metadata.DecodeBase64 == "true" {
```

Comment:
Can this be a boolean directly from Yaml?

https://yaml.org/type/bool.html

Reply:
Answered here: 

https://github.com/dapr/components-contrib/pull/251#pullrequestreview-369620175

Reply:
> > Only a small question.
> 
> Hello @artursouza . I tried that in the first place but the properties metadata signature is:
> 
> ```go
> type Metadata struct {
> 	Name       string
> 	Properties map[string]string `json:"properties"`
> }
> ```
> 
> So you get unmarshal error because of the map[string] string signature:
> 
> https://github.com/dapr/components-contrib/blob/master/bindings/metadata.go#L11
> 
> I've seen that other components are doing it the same way, maybe because of the properties field signature:
> 
> https://github.com/dapr/components-contrib/blob/ee97c3c56ada888e0740497131c2161f6a0a5539/secretstores/hashicorp/vault/vault.go#L121
> 
> cc @yaron2

That's correct, it's a `string[string]` map and thus has to be converted;

Reply:
yaron2. Then I think this is ok. I saw you already approved it.  Thanks!